### DISCUSSION TITLE
[SOLVED]RECOVERY ACCOUNT EMAIL IS NOT BEING SENT

### DISCUSSION THREAD

**Discussion by sinamum** [community/community#27669] [Other Feature Feedback, Questions, & Ideas] [answered]
My notebook broke.
I FOUND THE APP WHERE I CONFIGURED MY TWO FACTOR AUTHENTICATION, THANK COSMOS!

**Comment by jfullstackdev** [comment]
try here also 

https://support.github.com/contact/account

**Reply by sinamum** [reply]
I opened already, but I'm going to do another one.

**Comment by airtower-luna** [comment]
That as the message says the one-time password should be sent to your mail, so it's not related to 2FA.

> But this Recovery email is never sent to my email.

Check your spam folder, etc. And if the mail doesn't arrive after a little while, you can try the "resend email" link.

If none of that helps or you don't have access to the mail account any more, a support ticket as @jdevstatic mentioned is your only option.

**Reply by sinamum** [reply]
I'm clicking in this resend button and looking the spam about TWO DAYS.
IT IS NOT SENDING, and I have my email, I have my password, I  have my SSH key, I just need that the flow works.

**Comment by sinamum** [comment]
For explanation, I have my email, I have my password and I have SSH Key and I already open theTICKET, more than one, since wednesday, this post is to help to make them see my DESPAIR.
Please Github, I know I failed not doing the Two Factor Authentication, but that is a lesson learned, please, just help me. I'm going to open another ticket at support, I  opened about three already.  Some hour you need  to notice me.

**Reply by airtower-luna** [reply]
We other users can't help you, only support staff can. I'm sure you wouldn't want other random users to have access to your account. :confused: 

I recommend you do not open multiple tickets: I've seen similar posts where GitHub staff have described that the tickets then need to get consolidated (like, grouping your three tickets into one), which only slows the process down and sends you to the back of the queue again. I'm afraid you'll have to wait for support to get to your ticket.

**Reply by sinamum** [reply]
Ohh I see, I opened two, I confimed here.
So I'm going to open another one from this new account that I created and link this two other 🤔
And link this post too.
And ask for patience from cosmos right,  😬

**Reply by airtower-luna** [reply]
You should *not* open additional tickets if you want this resolved soon.

**Reply by sinamum** [reply]
ok :/
Just meditate and try to keep calm then.
Thanks for the tip 👌🏼

**Reply by airtower-luna** [reply]
One more thought: I haven't had to do account recovery myself so far, but for other support tickets I always got an automated "Request Received" mail very soon after creating them, with the hint that I can reply to the mail if I want to add anything. If you haven't received any such mail either, you could try to contact your email provider and ask if there's any issue with receiving mail from `github.com` and `githubsupport.com` mail addresses. :thinking:

**Reply by sinamum** [reply]
I received this ticket email response.
But thanks to remind me of that, maybe I can answer linking this and the ID of the other ticket righ.

**Comment by sinamum** [comment, marked as answer]
SOLVED, I had configured the two factor, then looking at my cellphone I found the app, Authy.
